modules/text_on_image/text_on_image_99_by_Jasmine_Davidson.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>In The City</title> \\Text on Image Animation
```

Comment:
Change the comment tag `\\Text on Image Animation` to `<!--Text on Image Animation-->`